.github/workflows/js_tests.yml

```diff
@@ -1,33 +1,14 @@
 name: JavaScript Testing
+
 on:
+  push:
+    branches:
+      - master
   pull_request:
-    paths:
-      - 'webpack/**'
-      - 'package.json'
-      - '.github/workflows/js_tests.yml'
```

Comment:
This would mean we want to run JS tests on every PR, is that correct?

Reply:
yes, I trusted https://github.com/theforeman/foreman_remote_execution/pull/964/files on this